<launch>

    
  <node name="urg_node_vertical" pkg="urg_node" type="urg_node" output="screen">
    <remap from="scan" to="scan_1" />
   <param name="ip_address" value=""/>
    <param name="serial_port" value="/dev/ttyACM1"/>
    <param name="frame_id" value="laser_1l"/>
    <param name="calibrate_time" value="true"/>
    <param name="publish_intensity" value="false"/>
    <param name="publish_multiecho" value="false"/>
    <param name="angle_min" value="-1"/>
    <param name="angle_max" value="1"/>
  </node>

 <node name="urg_node_horizontal" pkg="urg_node" type="urg_node" output="screen">
    <remap from="scan" to="scan_2" />
    <param name="ip_address" value="192.168.0.10"/>
    <param name="frame_id" value="laser_2"/>
    <param name="calibrate_time" value="true"/>
    <param name="publish_intensity" value="false"/>
    <param name="publish_multiecho" value="false"/>
    <param name="angle_min" value="-1"/>
    <param name="angle_max" value="1"/>
  </node>

 <node name="lidartocameralibrecorddata" pkg="lidartocameralibrecorddata" type="lidartocameralibrecorddata" output="screen">
  </node>
</launch>
